modules/using-assisted-installer-oci-agent-iso.adoc

```diff
@@ -6,62 +6,80 @@
 [id="using-assisted-installer-oci-agent-iso_{context}"]
 = Using the {ai-full} to generate an {oci}-compatible discovery ISO image
```

Comment:
Noticing this title is really long in the output. Would it work to omit AI and OCI since we are in the OCI via AI section anyway?
```suggestion
= Generating a compatible discovery ISO image
```